### DISCUSSION TITLE
Field Keywords On Details view

### DISCUSSION THREAD

**Discussion by CSWinnall** [cortexrd/Knack-Toolkit-Library#130] [Q&A] [answered]
Hi Norm

Do field keywords get triggered on a details view?

Craig

**Comment by cortexrd** [comment]
It really depends on each keyword.  Do you have a specific use case or an example?
For example, _cfv can be used in a details view.

Norm

**Reply by CSWinnall** [reply]
Hi sorry I understand what you are asking.  So there is no reason that you know of, that if a custom field keyword is used on a field in a details view, why it shouldn't trigger?

**Comment by CSWinnall** [comment]
I'm working on the tooltip function I can get it to trigger on form and
table but I don't get anything on a details view. I just wanted to check
first before I spend Hours trying to get it to work.  The keyword doesn't
trigger anything at all when the field is on the view.

C

**Reply by CSWinnall** [reply]
I am away at the moment I'll post what I'm trying to do when I get back.

**Comment by cortexrd** [comment]
You're right Craig.  Details views are not supported.  I've hit that same problem this morning, and fixing it now.
Will keep you updated.

Norm

**Reply by CSWinnall** [reply]
Thanks Norm.  I'll work on forms next and wait for your update.

**Comment by cortexrd** [comment, marked as answer]
Craig,

You can test the fix in pre-release **0.17.1**.

Norm

**Reply by CSWinnall** [reply]
Great thanks Norm.  I'll give it a go tomorrow when I'm back.

**Reply by CSWinnall** [reply]
@cortexrd 
Loving the _obf keyword will test all you have done (which is a lot) tomorrow.  Exciting times.
